README.md

```diff
@@ -14,8 +14,10 @@
 6. [TypeScript](#typescript)
 7. [CSS](#css)
    1. [Introduction](#introduction) 
-   2. [Disclaimer](#disclaimer)
-   3. [z-index](#z-index)
+   2. [Styleguide implementation](#styleguide-implementation)
+      1. [Colors](#colors)
```

Comment:
Weird that this one is 3 levels deep. Did you run `yarn generate-table-of-contents` to create this?

Reply:
No I wasn't aware of this. Will use from now on